### PR TITLE
New package: RazerInc.Synapse version 1.17.0.597

### PR DESCRIPTION
**Pull request by Teknoshark**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156431)

### CODE REVIEW

File: manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: RazerInc.Synapse
```
Comment (stephengillie) [inline manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.installer.yaml:4]:
```suggestion
PackageIdentifier: RazerInc.RazerInstaller.Beta
```
Please update file name and path to match this change.

File: manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RazerInc.Synapse
```
Comment (stephengillie) [inline manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: RazerInc.RazerInstaller.Beta
```
Please update file name and path to match this change.

File: manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: RazerInc.Synapse
```
Comment (stephengillie) [inline manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.yaml:4]:
```suggestion
PackageIdentifier: RazerInc.RazerInstaller.Beta
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
HI @Teknoshark,

Is this the beta version of  `RazerInc.RazerInstaller`? Should the `PackageIdentifier` be updated to match?

**Comment by Teknoshark** [comment]
Hi @stephengillie, I wasn't totally sure what to do about the version. Historically Razer has maintained major revisions of it's synapse software as separately maintained releases; currently the package maintained as `RazerInc.RazerInstaller` is Synapse 3, but Synapse 2 is still available and supported (just not through winget), and when this beta enters a release state, it will likely be maintained alongside Synapse 3 for some time, as opposed to clearly superseding it. Confounding things even more is the fact that for this beta, the major revision numbering scheme seems to have been dropped as well.  I'm happy to make the changes but was wondering if any of that mattered? Thanks for the help!

**Comment by stephengillie** [comment]
Thanks for the background. Ideally, `RazerInc.RazerInstaller` would be renamed to something like `RazerInc.Synapse.3`, and this could be similar to `RazerInc.Synapse.4.Beta`. What do you think?

As for dropping major revisions, how are they tracking progress? This `PackageIdentifier` could use an entirely different versionnig schema.

**Comment by stephengillie** [comment]
Thanks for taking the initiative to modify these. 

We have a limit of 1 manifest version change (add/modify/remove) per PR. If these changes could be split across probably 6 more PRs, then all should pass this check. (The rename has to be a deletion from one `PackageIdentifier` folder and an addition to the other. Modify can only occur within a `PackageIdentifier`, to my knowledge.)

**Pull request closed without merging**